pkg/operator/ceph/csi/controller.go

```diff
@@ -263,3 +272,62 @@ func (r *ReconcileCSI) reconcile(request reconcile.Request) (reconcile.Result, e
 
 	return reconcileResult, nil
 }
+
+func reportNodeTelemetry(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo) {
```

Comment:
Instead of creating this method during the csi reconcile, we can just implement it inside the existing [reportTelemetry](https://github.com/rook/rook/blob/a64f9bccf73cbb91dfd87e129255b0df47101436/pkg/operator/ceph/cluster/cluster.go#L524) method. Then we know the ceph cluster connection is available. I don't see a need to keep it with csi, especially since not all these node metrics are specific to csi.